src/databases/coursesDb.js

```diff
@@ -55,10 +91,23 @@ const addCourse = async ({
   creatorId,
 }) => {
   const trx = await knex.transaction();
-  newCourse({ trx, name, description })
-    .then(() => createCourseCreator({ trx, courseId, creatorId }));
+  await newCourse({
+    trx,
+    name,
+    description,
+    courseId,
+  });
+  await createCourseCreator({
```

Comment:
usually the transactions are managed in the services and there exist one database/X.js per table, but it is ok for this case ;)

Reply:
Excellent, I'll do that. I didn't know whether transactions could be managed in the service layer or it was something related to databases.